extension/js/common/core/attachment.ts

```diff
@@ -16,7 +16,8 @@ export type FcAttachmentLinkData = { name: string, type: string, size: number };
 
 export class Attachment {
 
-  public static readonly webmailNamePattern = /^(((cryptup|flowcrypt)-backup-[a-z0-9]+\.(key|asc))|(.+\.pgp)|(.+\.gpg)|(.+\.asc)|(noname)|(message)|(PGPMIME version identification)|())$/m;
+  // eslint-disable-next-line max-len
+  public static readonly webmailNamePattern = /^(((cryptup|flowcrypt)-backup-[a-z0-9]+\.(key|asc))|(.+\.pgp)|(.+\.gpg)|(.+\.asc)|(noname)|(message)|(PGPMIME version identification)|(ATT[0-9]{5})|())$/m;
```

Comment:
Looks like we actually need these patterns only to trigger message download in gmail replacer. This makes the application somewhat unreliable as the PGP/MIME format doesn't require for specific filenames to be present. This means we are prone to encountering similar issues in future.
@tomholub Should we consider trying to download GmailMsg for every message (or based on some other criteria) at least to check for `Content-Type: multipart/encrypted` etc.?

Reply:
@tomholub this triggering mechanism doesn't seem very reliable for PGP/MIME messages

Reply:
ping @tomholub

Reply:
Agree, will file an issue - not for this PR.